### PR TITLE
docs: add dbms research documentation

### PR DESCRIPTION
**Pull request by gukj-spel**
pr for issue #38 , Database Survey: Add a table describing the survey results

### CODE REVIEW

File: src/dbtest/src/doc/Database_Resarch_Doc.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### 主流 DMBS 支持情况
```
Comment (hldgs) [inline src/dbtest/src/doc/Database_Resarch_Doc.md:1]: please translate it to english, and can you add "support isolation level" column to describe isolation levels supported by each dbms? And can you add "transaction descriptions" column to describe their transaction implementation, such as mvcc or 2pl, optimism or pessimism?

**Comment by hldgs** [comment]
please translate your pr description to english, and use this [format](https://www.conventionalcommits.org/zh-hans/v1.0.0/#%e7%ba%a6%e5%ae%9a%e5%bc%8f%e6%8f%90%e4%ba%a4%e8%a7%84%e8%8c%83) to title your pr

**Comment by gukj-spel** [comment]
> please translate your pr description to english, and use this [format](https://www.conventionalcommits.org/zh-hans/v1.0.0/#%e7%ba%a6%e5%ae%9a%e5%bc%8f%e6%8f%90%e4%ba%a4%e8%a7%84%e8%8c%83) to title your pr

ok

File: src/dbtest/src/doc/Database_Resarch_Doc.md (hunk 1 above)
Comment (gukj-spel) [reply src/dbtest/src/doc/Database_Resarch_Doc.md:1]: ok, sure.

**Pull request merged**